modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request_resolution.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequestResolution < ApplicationRecord
+    belongs_to :power_of_attorney_request,
+               class_name: 'AccreditedRepresentativePortal::PowerOfAttorneyRequest',
+               inverse_of: :resolution
+
+    RESOLVING_TYPES = [
+      'AccreditedRepresentativePortal::PowerOfAttorneyRequestExpiration',
+      'AccreditedRepresentativePortal::PowerOfAttorneyRequestDecision'
+    ].freeze
+
+    delegated_type :resolving, types: RESOLVING_TYPES, dependent: :destroy, optional: true
+
+    has_kms_key
+
+    has_encrypted :reason, key: :kms_key, **lockbox_options
+
+    # Validations
+    validates :resolving_type, presence: true, inclusion: { in: RESOLVING_TYPES, allow_nil: true }
+    validates :resolving_id, presence: true, if: -> { resolving_type.present? }
+
+    module Resolving
+      extend ActiveSupport::Concern
+
+      included do
+        has_one(
+          :power_of_attorney_request_resolution,
+          as: :resolving,
+          touch: true
```

Comment:
looked up the `touch` option and it's for updated timestamps which the sub types don't have so we can remove this